Redlight.py

```diff
@@ -171,11 +181,13 @@ def compare(im, roles):
 
         #SCANS FOR PARASAUR PING (CYAN) ALERT = 7
         if (0, 255, 234) in lop2:
-            logger.error("Alert Found")
+            logger.error("Alert Found: Parasaur Ping Detected")
 
             #SETS UP MESSAGE FOR DISCORD WEBHOOK IF ALERT FOUND
             content = (f"{roles} - **Parasaur, Simply Too Close** - {who}")
-            return LiveHook(content)
+            #TODO: Fix config such that true/false are parsed as boolean rather than strings
+            if parasaur_ping_alert_enabled == 'true':
+                return LiveHook(content)

```

Comment:
This is where I'm conditionally sending the webhook depending on the config value.